e2e-tests/functions

```diff
@@ -12,6 +12,7 @@ deploy_operator() {
 	yq eval \
 		"$(printf 'select(documentIndex==0).spec.template.spec.containers[0].image="%s"' "${IMAGE}")" \
 		"${DEPLOY_DIR}/operator.yaml" \
+		| yq eval '.spec.template.spec.containers[0].env += {"name": "PGO_FEATURE_GATES", "value": "InstanceSidecars=true"}' - \
```

Comment:
@egegunes please enable it by default

Reply:
https://github.com/percona/percona-postgresql-operator/pull/313/commits/422611dd74c87af724c7bf38f628e6c5286f291a